doc/examples/registration/plot_stitching.py

```diff
@@ -0,0 +1,160 @@
+"""
+======================
+Simple image stitching
+======================
+
+This example demonstrates how a set of images can be assembled under
+the hypothesis of rigid body motions.
+
+"""
+
+from matplotlib import pyplot as plt
+import numpy as np
+from skimage.data import moon
+from skimage.util import random_noise, img_as_float
+from skimage.transform import warp, EuclideanTransform, rotate
+from skimage.feature import corner_harris, corner_peaks
+from skimage.measure import ransac
+from skimage.filters import gaussian
+from skimage.metrics import peak_signal_noise_ratio
+
+
+def match_locations(img0, img1, coords0, coords1, radius=5, sigma=3):
+    """Image locations matching using SSD minimization.
```

Comment:
```suggestion
    """Match image locations using SSD minimization.
```
Reference: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings